backend/lib/edgehog/deployment_campaigns/deployment_mechanism/lazy/executor.ex

```diff
@@ -107,12 +115,15 @@ defmodule Edgehog.DeploymentCampaigns.DeploymentMechanism.Lazy.Executor do
       tenant_id
       |> Core.get_deployment_campaign!(deployment_campaign_id)
       |> Ash.load!(
-        [release: [containers: [:networks, :volumes, :image]]],
+        [:target_release, release: [containers: [:networks, :volumes, :image]]],
```

Comment:
```suggestion
        [target_release: [], release: [containers: [:networks, :volumes, :image]]],
```
(ash would otherwise complain at runtime)